### PR TITLE
Add intial BridgeDropdown

### PR DESCRIPTION
**Pull request by alexkeating**
Pr to resolve issue #150

### CODE REVIEW

File: packages/react-app/src/components/common/BridgeDropdown.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+export const BridgeDropdown = () => {
+  const { providerChainId: chainId } = useContext(Web3Context);
+  const bridgeChainId = getBridgeNetwork(chainId);
+  const defaultBridgeUrl = getBridgeUrl(`${chainId}-${bridgeChainId}`);
+  const defaultBridgeName = getBridgeName(defaultBridgeUrl);
+  const [bridgeName, setBridgeName] = useState(defaultBridgeName);
+
+  const changeItem = e => {
+    setBridgeName(e.target.name);
+    window.location.href = e.target.dataset.url;
```
Comment (alexkeating) [inline packages/react-app/src/components/common/BridgeDropdown.jsx:17]: Is the end goal of this issue to redirect here? One question I have is how do we want to handle the different urls when developing locally?

**Comment by dan13ram** [comment]
Hey @alexkeating, Thanks for giving this a shot!

The idea here is to support different bridges in the **same** UI by choosing from the drop-down, and not redirect to a separate URL.

Currently we are fetching the HOME_CHAIN_ID and FOREIGN_CHAIN_ID from the env, but we should create a way to choose the bridge-type ['mainnet-xdai', 'bsc-xdai', 'kovan-sokol'] and set the home and foreign chain Ids globally for the app.

We can perhaps have this list of bridge-types configurable in the env and support ['mainnet-xdai', 'bsc-xdai'] in production and have ['mainnet-xdai', 'bsc-xdai', 'kovan-sokol'] in staging.

**Comment by alexkeating** [comment]
Oh okay that makes sense the issue was a little ambiguous. I will update my pr!

**Comment by maxaleks** [comment]
Hi @alexkeating!
Thanks for contribution!
We discussed this task with our team and I'll provide you some details.

You should create a config file (for example, `networks.js`) with the following content:
```javascript
export const networks = {
  'mainnet-xdai': {
    homeChainId: 100,
    foreignChainId: 1,
    enableReversedBridge: false,
    gasPriceSupplierUrl: 'https://gasprice.poa.network/',
    gasPriceSpeedType: 'fast',
    gasPriceUpdateInterval: 60000,
    ethPriceApiUrl: 'https://api.coingecko.com/api/v3/simple/price?ids=ethereum&vs_currencies=USD',
    ethPriceUpdateInterval: 15000,
    foreignMediatorAddress: '0x88ad09518695c6c3712AC10a214bE5109a655671',
    homeMediatorAddress: '0xf6A78083ca3e2a662D6dd1703c939c8aCE2e268d',
    foreignAmbAddress: '0x4C36d2919e407f0Cc2Ee3c993ccF8ac26d9CE64e',
    homeAmbAddress: '0x75Df5AF045d91108662D8080fD1FEFAd6aA0bb59',
    foreignGraphName: 'raid-guild/mainnet-omnibridge',
    homeGraphName: 'raid-guild/xdai-omnibridge',
    homeRpcUrl: process.env.REACT_APP_XDAI_RPC_URL,
    homeExplorerPrefix: 'https://blockscout.com/xdai/mainnet',
    foreignRpcUrl: process.env.REACT_APP_MAINNET_RPC_URL,
    foreignExplorerPrefix: 'https://blockscout.com/eth/mainnet',
    ambLiveMonitorPrefix: 'https://alm-xdai.herokuapp.com',
  },
  'bsc-xdai': {
    homeChainId: 100,
    foreignChainId: 56,
    enableReversedBridge: true,
    foreignMediatorAddress: '0xF0b456250DC9990662a6F25808cC74A6d1131Ea9',
    homeMediatorAddress: '0x59447362798334d3485c64D1e4870Fde2DDC0d75',
    foreignAmbAddress: '0x05185872898b6f94AA600177EF41B9334B1FA48B',
    homeAmbAddress: '0x162E898bD0aacB578C8D5F8d6ca588c13d2A383F',
    foreignGraphName: 'maxaleks/bsc-to-xdai-omnibridge',
    homeGraphName: 'maxaleks/xdai-to-bsc-omnibridge',
    homeRpcUrl: process.env.REACT_APP_XDAI_RPC_URL,
    homeExplorerPrefix: 'https://blockscout.com/xdai/mainnet',
    foreignRpcUrl: process.env.REACT_APP_BSC_RPC_URL,
    foreignExplorerPrefix: 'https://bscscan.com',
    ambLiveMonitorPrefix: 'https://alm-bsc-xdai.herokuapp.com',
  },
  'kovan-sokol': {
    homeChainId: 77,
    foreignChainId: 42,
    enableReversedBridge: true,
    foreignMediatorAddress: '0xA960d095470f7509955d5402e36d9DB984B5C8E2',
    homeMediatorAddress: '0x40CdfF886715A4012fAD0219D15C98bB149AeF0e',
    foreignAmbAddress: '0xFe446bEF1DbF7AFE24E81e05BC8B271C1BA9a560',
    homeAmbAddress: '0xFe446bEF1DbF7AFE24E81e05BC8B271C1BA9a560',
    foreignGraphName: 'dan13ram/kovan-omnibridge',
    homeGraphName: 'dan13ram/sokol-omnibridge',
    homeRpcUrl: process.env.REACT_APP_SOKOL_RPC_URL,
    homeExplorerPrefix: 'https://blockscout.com/poa/sokol',
    foreignRpcUrl: process.env.REACT_APP_KOVAN_RPC_URL,
    foreignExplorerPrefix: 'https://blockscout.com/eth/kovan',
    ambLiveMonitorPrefix: 'https://alm-test-amb.herokuapp.com',
  },
};
```
And then you should update the code to use params of the selected bridge type instead of environment variables.

Please let us know if you will be working on this task. Thanks!

**Comment by alexkeating** [comment]
^ Thanks, I am planning to take a look at this tonight

**Comment by dan13ram** [comment]
Closing this and replacing with #190

**Pull request closed without merging**